### PR TITLE
Feature/loggers metadata and flags def

### PR DESCRIPTION
**Pull request by ZofiaRudjord**

### CODE REVIEW

**Comment by ZofiaRudjord** [comment]
I added 2 new sensorcode_cv to be able to store infomation that a given time series measurement value was Approved or Discarded by domain expert.

File: src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py [hunk 1]
```diff
@@ -6,9 +6,28 @@
     EquipmentModelCreate,
     LoggersTimeSeriesCreate,
     DataQualityCreate,
+    ControlledVocabularyCreate,
 )
 
 
+def loggers_controlled_vocabularies() -> List[ControlledVocabularyCreate]:
```
Comment (knl88) [inline src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:13]: Guess we should reference the CV used

**Review by knl88 — APPROVED** [review]
lgtm

**Review by jemmima — COMMENTED** [review]
LGTM

**Pull request merged**